tokio/src/sync/mpsc/bounded.rs

```diff
@@ -817,3 +975,120 @@ impl<T> fmt::Debug for Permit<'_, T> {
             .finish()
     }
 }
+
+// ===== impl Permit =====
+
+impl<T> OwnedPermit<T> {
+    /// Sends a value using the reserved capacity.
+    ///
+    /// Capacity for the message has already been reserved. The message is sent
+    /// to the receiver and the permit is consumed. The operation will succeed
+    /// even if the receiver half has been closed. See [`Receiver::close`] for
+    /// more details on performing a clean shutdown.
```

Comment:
The documentation for `Receiver::close` links to `Permit` in relation to this, but should probably link to `OwnedPermit` too.